Fix overlapping items in directory permission checks in installation wizard

luismrsilva opened the pull request:
**Explanation:**
Firefox 75.0 states that, regarding `display: table-column` in `.formColumnLeft` and `.formColumnRight`: 

> The **display** value has been changed by the engine to **block** because the element is **floated**.
> Try removing **float** or adding **display: block**.

Also, regarding `width: 100%` in `.formRow`:
> **width** has no effect on this element since it has a display of **table-row**.
> Try adding **display:inline-block** or **display:block**.

Which means there are "contradicting CSS properties".

**What I did:**
- Remove `display: table-column` from affected elements (`.formColumnLeft` and `.formColumnRight`), as these are floated. Also removed it from `.formColumn`, although it is not used in the page mentioned in the issue.
- Remove `display: table-row` from `.formRow`, so that `width:100%` applies.
- Add `overflow: hidden` to `.formRow`, which contains floating elements, so that its [height is not zero](https://stackoverflow.com/questions/9024494/css-floating-divs-have-0-height) and the background appears.

Use of "display: table-*" should probably be removed elsewhere, especially in floated elements. However, this is enough to fix this specific issue (#3449).

**Also, I did not check if it breaks things in other pages.**

Here's the result:
![Screenshot from 2020-04-19 04-12-23](https://user-images.githubusercontent.com/6828187/79678524-5a95ee00-81f4-11ea-8daf-fd177c63da82.png)

luismrsilva committed:
Fix overlapping itens in directory permission checks in installation wizard

Issue #3449.

TheWitness commented:
How does this affect form displays after install?

nuno-silva commented:
Here's some other forms **after** the patch:

-  in the installation wizard: ![image](https://user-images.githubusercontent.com/6935057/79692618-1fc8a000-825e-11ea-9b15-b14214e09aad.png)

- New Device form:
![image](https://user-images.githubusercontent.com/6935057/79692912-c06b8f80-825f-11ea-9d76-eb47c65ef672.png)


- Automation -> Networks -> edit network:
![image](https://user-images.githubusercontent.com/6935057/79692973-058fc180-8260-11ea-8014-a97540582b5b.png)

I can provide more screenshots if you have a particular request.

TheWitness commented:
Okay, can you please update the CHANGELOG, and then I will merge.  Thanks!

luismrsilva committed:
Update CHANGELOG with issue #3449

Also, use "overflow-y: auto" instead of "overflow: hidden", which gets the job done (fixes height being 0 when element only contains floated elements), but is safer (won't hide things).
